tests/regression/00-sanity/33-hoare-over-paths.c

```diff
@@ -27,8 +27,8 @@ int main() {
   // NEW explanation:
   // using SensitiveDomain correctly keeps both paths as path-sensitivity demands
 
-  // TODO: manually check final join node in HTML
-  // cannot be automated because concrete execution cannot check
+  // cram test checks internal result
```

Comment:
Would this not need a `CRAM` annotation? Or does it randomly match because the matching is case-insensitive?

Reply:
It includes asserts, so there are still some checks for update_suite. But I also added the `CRAM` annotation now. It shouldn't be case-insensitive.